src/events/balancer.py

```diff
@@ -199,8 +199,10 @@ def rebalance(self, event: ActionEvent) -> None:
         if self.charm.state.runs_broker:
             available_brokers = [broker.unit_id for broker in self.charm.state.brokers]
         else:
-            brokers: list = self.charm.state.balancer.broker_capacities.get("brokerCapacities", [])
-            available_brokers = [int(broker["brokerId"]) for broker in brokers]
+            brokers: list = [
```

Comment:
**todo/nitpick:** Unless I'm mistaken, I believe this typehint is redundant.

Reply:
Removed as part of a6e2efbc3484